web/book/src/SUMMARY.md

```diff
@@ -74,7 +67,7 @@
 
 - [Integrations](./integrations/README.md)
 
-  - [CLI](./integrations/cli.md)
+  - [`prqlc`](./integrations/prqlc.md)
```

Comment:
What should this be? I think it is / will be referred to as `prqlc`, but maybe it should be more obvious for folks who are less used to compilers?

Reply:
"prqlc CLI"?

Reply:
Funny you should suggest, check the latest :)